tests/braille-specs/syc.yaml

```diff
@@ -0,0 +1,38 @@
+
+# Yaml Test For Syriac
+#
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+display: unicode.dis
+table:
+  language: syc
+  grade: 1
+  __assert-match: syc.tbl
+flags: { testmode: forward }
+tests:
+  # Syriac
+  - ["ܩܦ̄ܐ", "⠟⠏̄⠁"]
```

Comment:
I have questions about this box character in the expected result. it looks like the line over the middle letter is its own unicode character that maybe isn't in your table yet?

Reply:
We have been looking in to this but we aren't sure what character you are refering to. Could you reply and put the char in your comment please. As far as I can tell all of our characters are represented by braille in the tests.

Reply:
<img width="170" alt="image" src="https://github.com/liblouis/liblouis/assets/6843203/0c26ae4c-2288-4c69-b221-149f3796b24a">
Maybe the font isn't rendering on my side? I don't know. I'm seeing that the first item in the array is some incredible Syriac text, and the second item in the array is braille dots 12345, then a box with a line over it, then braille dot 1.  

When I copy-paste it, it comes out looking even weirder:
<img width="58" alt="image" src="https://github.com/liblouis/liblouis/assets/6843203/3fa38b1b-ed2a-45a1-ba1b-d13fa00c30cf">


⠟⠏̄⠁

Reply:
it seems like there's a "macron" diacritical mark  U+0304 attached to the unicode braille character for dots 1235,  U+280F.

I only noticed it because it renders weird in my browser. it renders weird, presumably, because unicode representations for braille dots are not meant to have diacritic markers attached to them. 

I'm guessing that the "macron" diacritical mark is in the input, but is not defined in the braille table, so it didn't get transformed into braille, and whatever program you used to generate these tests and their expected output didn't notice that an undefined character slipped through. 

I think what that means is that neither liblouis nor the code that is generating your test cases is checking to make sure that the output only contains braille characters.

Reply:
It looks like the input for this test, in unicode, is 
\u0729\u0726\u0304\u0710 and the output is \u281f\u280f\u0304\u2801. The problem is that the `\u0304` slipped through without being transformed.

I don't know anything about Syriac, but it seems like this "\u0304" macron character (it looks like a jaunty hat that floats above a letter) means something in Syriac.  So I think your tests inadvertently caught a problem in your table, or they would have, if the expected test output hadn't been automatically generated. so really I think we've found two problems here:
1) that your test cases are being auto-generated (which means that a passing test doesn't necessarily indicate that the table is generating the correct output), and 
2) that this macron character is missing from the Syriac table.